vuepress/docs/next/tutorials/create/ms/run-local.md

```diff
@@ -143,18 +81,19 @@ By default these variables are set to:
     REACT_APP_KEYCLOAK_REALM=jhipster
     REACT_APP_KEYCLOAK_CLIENT_ID=web_app
 
-2. By default, Docker Compose recreates the Keycloak container and resets the H2 database each time it is started. To persist Keycloak changes across restarts, you can make the following updates: 
-   * In your src/main/docker/keycloak.yml file, replace\
-   `
-   '-Dkeycloak.migration.strategy=OVERWRITE_EXISTING',` with 
-   ```
-   '-Dkeycloak.migration.strategy=IGNORE_EXISTING',
-   ```   
-   * In the same file, add a persistent volume under `volumes`:
+2. In this Blueprint generated project, Docker Compose persists Keycloak data across restarts by default. If you want your data to reset on restarts:
```

Comment:
@nshaw, I think the default got reversed, please check that this section is correct

Reply:
@jyunmitch agreed. At one point I requested that the default be changed since it makes more sense to key data across restarts. Anthony won't be changing this part so please reword to match the new style. I confirmed the logic was switched (a long time ago actually) - https://github.com/entando/generator-jhipster-entando/blob/move-to-v5-descriptor/generators/server/templates/src/main/docker/keycloak.yml.ejs.